diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9c01d0c..b9b4254 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,10 +28,6 @@ if(${JWT_SSL_LIBRARY} MATCHES "LibreSSL")
   find_package(LibreSSL 3.0.0 REQUIRED)
 endif()
 
-if(JWT_EXTERNAL_PICOJSON)
-  find_package(picojson 1.3.0 REQUIRED)
-endif()
-
 set(JWT_INCLUDE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/include)
 set(JWT_HEADER_FILES ${JWT_INCLUDE_PATH}/jwt-cpp/jwt.h)
 if(NOT JWT_DISABLE_BASE64)
