From baeb80dd8c7af356c536bba73f8b31d0255416e3 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Wed, 5 May 2021 12:26:05 +0100
Subject: [PATCH 2/2] disable cmake compiler checks

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 695bbf0..c7c7714 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ if(HUNTER_ENABLED)
   message(STATUS "jwt-cpp: using hunter for dependency resolution")
 endif()
 
-project(jwt-cpp)
+project(jwt-cpp NONE)
 
 option(JWT_BUILD_EXAMPLES "Configure CMake to build examples (or not)" ON)
 option(JWT_BUILD_TESTS "Configure CMake to build tests (or not)" OFF)
